src/app/Processor/ProcessorDetail/ProcessorDetail.css

```diff
@@ -0,0 +1,4 @@
+.processor-detail__filters,
+.processor-detail__transformation-template {
+  max-width: 800px;
```

Comment:
why `800px`? do we have some PF variable that may fit this case? Or in the worst scenario, can we use `em` or `rem`?

Reply:
I've taken a second look at PF about this. There's only a `isWidthLimited` prop for page sections, adding the `pf-m-limit-width` class. But the limit is super wide, ~100 rem. So not so useful for us.

Converted the value to `rem` by the way.